test/ssc_test/cmod_windpower_test.cpp

```diff
@@ -132,10 +132,90 @@ TEST_F(CMWindPowerIntegration, WakeModelsUsingFile_cmod_windpower) {
+    ssc_data_set_number(data, "wake_loss_multiplier", 1.0);
+    compute();
+    ssc_data_get_number(data, "annual_wake_loss_internal_percent", &withoutMultiplier);
+    ssc_data_set_number(data, "wake_loss_multiplier", multiplier);
+    compute();
+    ssc_data_get_number(data, "annual_wake_loss_internal_percent", &withMultiplier);
+    if (withoutMultiplier != 0.)
+        EXPECT_NEAR((withMultiplier / withoutMultiplier), multiplier, 0.01) << "EV Wake Model Multiplier";
+
+    //Constant Loss model
+    ssc_data_set_number(data, "wind_farm_wake_model", 3);
+    ssc_data_set_number(data, "wake_int_loss", 5);
+    ssc_data_set_number(data, "wake_loss_multiplier", 1.0);
+    compute();
+    ssc_data_get_number(data, "annual_wake_loss_total_percent", &withoutMultiplier);
+    ssc_data_set_number(data, "wake_loss_multiplier", multiplier);
+    compute();
+    ssc_data_get_number(data, "annual_wake_loss_total_percent", &withMultiplier);
+    if (withoutMultiplier != 0.)
+        EXPECT_NEAR((withMultiplier / withoutMultiplier), multiplier, 0.01) << "Constant Loss Wake Model Multiplier";
+
+
+}
+
+/// Test using the optional coefficient of thrust curve input for the park model
+TEST_F(CMWindPowerIntegration, CtCurve_cmod_windpower)
+{
+    //set the park wake model and get original wake loss
+    ssc_data_set_number(data, "wind_farm_wake_model", 1); 
+    compute();
+    ssc_number_t wakeLoss1, wakeLoss2;
+    ssc_data_get_number(data, "annual_wake_loss_internal_percent", &wakeLoss1); //get the wake loss without setting Ct curve
+
+    // use a bogus ct curve just to test that it does something
+    ssc_number_t ctCurve[161] = { 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5,
+        0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5,
+        0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5,
+        0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5,
+        0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5,
+        0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5 };
+    ssc_data_set_array(data, "wind_turbine_ct_curve", ctCurve, 161);
+        
+    // get new wake loss
+    compute();
+    ssc_data_get_number(data, "annual_wake_loss_internal_percent", &wakeLoss2); //get the wake loss with Ct curve
+
+    ssc_number_t difference = wakeLoss2 - wakeLoss1;
+    EXPECT_NEAR(wakeLoss1, 4.148, 0.01) << "Wake Loss 1";
+    EXPECT_NEAR(wakeLoss2, 5.562, 0.5) << "Wake Loss 2";
+    EXPECT_NEAR(difference, 1.413, 0.5) << "Difference";
+}
+
```

Comment:
Can you please edit "IcingAndLowTempCutoff_cmod_windpower" test to add the icing persistence feature?

Reply:
I had written this test, but hadn't pushed it. Done!